extension/src/popup/components/sendPayment/SendConfirm/TransactionDetails/index.tsx

```diff
@@ -172,47 +234,61 @@ export const TransactionDetails = ({ goBack }: { goBack: () => void }) => {
     })();
   }, [destAsset.code, destAsset.issuer, networkDetails]);
 
-  const computeDestMinWithSlippage = (
-    slippage: string,
-    destMin: string,
-  ): BigNumber => {
-    const mult = 1 - parseFloat(slippage) / 100;
-    return new BigNumber(destMin).times(new BigNumber(mult));
-  };
+  useEffect(() => {
+    dispatch(getBlockedAccounts());
+  }, [dispatch]);
 
-  const getOperation = () => {
-    // path payment or swap
-    if (isPathPayment || isSwap) {
-      const destMin = computeDestMinWithSlippage(
-        allowedSlippage,
-        destinationAmount,
-      );
-      return StellarSdk.Operation.pathPaymentStrictSend({
-        sendAsset: sourceAsset,
-        sendAmount: amount,
-        destination: isSwap ? publicKey : destination,
+  // checked tags for blocked accounts
+  useEffect(() => {
+    if (isValidatingSafeAssetsEnabled) {
```

Comment:
Ah, this is actually the incorrect setting to check. It should be `isValidatingSafetyValue`. (This naming is pretty confusing!)

`isValidatingSafeAssetsEnabled` is a toggle used to check  if a trustline is safe to be added

Reply:
These lines might be a good model to use here: https://github.com/stellar/freighter/blob/master/extension/src/background/messageListener/freighterApiMessageListener.ts#L127-L129

Also, we probably only want to do this check on Mainnet as the mainnet account is actually the one that's flagged

Reply:
thanks, I corrected this in 8dea16edd6ef4a38d6795e0ce204f909df6414bc